paperqa/clients/__init__.py

```diff
@@ -19,19 +19,14 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_CLIENTS: (
```

Comment:
We're losing some semantics here it takes either a collection OR a sequence of collections -- this is removing the second possibility.

Reply:
So the place this gets used still has both. Imo it's fine for the default to only have one of the logic branches

Reply:
Ahh i see what you're saying I was thinking this was in a doc signature -- makes sense.